startup/81-dexela.py

```diff
@@ -0,0 +1,69 @@
+import os
+import ophyd
+from hxntools.detectors.dexela import (DexelaDetector, DexelaTiffPlugin)
+from ophyd import (AreaDetector, CamBase, TIFFPlugin, Component as Cpt,
+                    HDF5Plugin, Device, StatsPlugin, ProcessPlugin,
+                    ROIPlugin, EpicsSignal)
+from databroker.assets.handlers import HandlerBase
+from ophyd.areadetector.filestore_mixins import (FileStoreIterativeWrite,
+                                                 FileStoreHDF5IterativeWrite,
+                                                 FileStoreTIFFSquashing,
+                                                 FileStoreTIFF,
+                                                 FileStoreHDF5, new_short_uid,
+                                                 FileStoreBase
+                                                 )
+from ophyd.areadetector import (AreaDetector, PixiradDetectorCam, ImagePlugin,
+                                TIFFPlugin, StatsPlugin, HDF5Plugin,
+                                ProcessPlugin, ROIPlugin, TransformPlugin,
+                                OverlayPlugin)
+from ophyd.areadetector.trigger_mixins import SingleTrigger
+from enum import Enum
+
+
+class XPDDMode(Enum):
+    step = 1
+    fly = 2
+
+
+class XPDDDexelaDetector(SingleTrigger, DexelaDetector):
+    total_points = Cpt(Signal, value=1, doc="The total number of points to be taken")
+    tiff1 = Cpt(DexelaTiffPlugin, 'TIFF1:',
+               read_attrs=[],
+               configuration_attrs=[],
+               write_path_template='Z:%Y\\%m\\%d\\',
```

Comment:
```suggestion
               write_path_template='Z:\\dex_data\\%Y\\%m\\%d\\',
```

Reply:
I think this way it will work. Maybe `dex_data` is not available there, try checking the Windows IOC to get the correct path template.

Reply:
It's an outdated suggestion, which cannot be applied. Will submit a new one.